src/walletSdk/Horizon/Transaction/TransactionBuilder.ts

```diff
@@ -90,6 +90,35 @@ export class TransactionBuilder extends CommonTransactionBuilder<TransactionBuil
     return this;
   }
 
+  pathPay(
+    destinationAddress: string,
+    sendAsset: StellarAssetId,
+    destAsset: StellarAssetId,
+    amount: string,
+    destMin?: string,
+  ): TransactionBuilder {
+    this.operations.push(
+      StellarSdk.Operation.pathPaymentStrictSend({
```

Comment:
Do we not support strictReceive at the time?

Reply:
commented above: https://github.com/stellar/typescript-wallet-sdk/pull/56#discussion_r1324892719